mocca-client/src/main/java/com/paypal/mocca/client/MoccaFeignEncoder.java

```diff
@@ -29,6 +34,16 @@
 class MoccaFeignEncoder implements Encoder {
 
     private final MoccaSerializer moccaSerializer = new MoccaSerializer();
+    Validator validator;
```

Comment:
Can we make this private? Or is this a POC type of code?

Reply:
I can make it private.